cluster/ci/monitoring/mixins/grafana_dashboards/boskos.jsonnet

```diff
@@ -75,6 +75,10 @@ dashboard.new(
     .addTarget(prometheus.target(
         'sum(kube_pod_container_status_running{pod=~".*-openstack(-.*)?",container="teardown"})',
         legendFormat='openstack',
+    ))
+    .addTarget(prometheus.target(
+        'sum(kube_pod_container_status_running{pod=~".*-openstack-vexxhost(-.*)?",container="teardown"})',
+        legendFormat='openstack-vexxhost',
```

Comment:
The way I read this is that the existing target above will also include `openstack-vexxhost` and this new one is basically just a more specialized view. Is that intended?

Reply:
Oh indeed. No, not intended. This is a completely a separate cloud than `openstack`.

Maybe we can rework the `(-.*)?` part of the openstack regex above to match more specifically the expected suffix? Do you know what we're trying to match exactly?

Reply:
I'm not 100% familiar with all this, but I think we are matching pods names against that regeq, and only count those pods that contain a teardown container. This is how the pods created by ci-operator end-to-end tests look like (mostly by convention) (@hongkailiu can you please confirm the above -- you were the one setting this up, right?)

This also expects the tests in ci-op configs to be named accordingly, because names of the pods follow the names of the tests.

Reply:
So for now, I'm just going to add the OpenStack Vexxhost panel without differentiating between MOC or Vexxhost in the "Running clusters by platform" panel.